### PR TITLE
Prefer emulators:start over serve

### PR DESCRIPTION
**Pull request by samtstern**
### Description

Two changes:

  * Update functions templates to use `emulators:start` instead of `serve`
  * Warn folks who start either db emulator to use the new command.
	 
### Scenarios Tested

```
 npx firebase serve --only firestore
⚠  firebase emulators:start way to start the Realtime Database or Cloud Firestore emulators.
i  firestore: Serving ALL traffic (including WebChannel) on http://localhost:8080
⚠  firestore: Support for WebChannel on a separate port (8081) is DEPRECATED and will go away soon. Please use port above instead.
i  firestore: Emulator logging to firestore-debug.log
✔  firestore: Emulator started at http://localhost:8080
```

### Sample Commands

N/A

### CODE REVIEW

**Comment by samtstern** [comment]
Bumping this one back into your inbox.

File: src/commands/serve.js [hunk 1]
```diff
@@ -58,6 +58,14 @@ module.exports = new Command("serve")
   })
   .action((options) => {
     options.targets = filterOnly(VALID_EMULATORS, options.only);
+    if (options.targets.indexOf("database") >= 0 || options.targets.indexOf("firestore") >= 0) {
+      utils.logWarning(
+        `${clc.bold(
+          "firebase emulators:start"
+        )} is the preferred way to start the Realtime Database or Cloud Firestore emulators.`
```
Comment (yuchenshi) [inline src/commands/serve.js]: Please the wording actionable, like: "Please use `firebase emulators:start` instead to start ..."

Also question: are we deprecating / removing `serve --only`?

File: src/commands/serve.js [hunk 2]
```diff
@@ -58,6 +58,14 @@ module.exports = new Command("serve")
   })
   .action((options) => {
     options.targets = filterOnly(VALID_EMULATORS, options.only);
+    if (options.targets.indexOf("database") >= 0 || options.targets.indexOf("firestore") >= 0) {
```
Comment (bkendall) [inline src/commands/serve.js]:
```suggestion
    if (options.targets.includes("database") || options.targets.includes("firestore")) {
```

**Comment by samtstern** [comment]
@bkendall yeah I changed the docs a while back.

File: src/commands/serve.js (hunk 1 above)
Comment (samtstern) [reply src/commands/serve.js]: In 8.0 we will almost definitely remove `database` and `firestore` from `serve` and make it a Hosting/Functions thing only (and even for Functions we will still warn people it's not the best way)

**Pull request merged**